src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Colors.values()[getRandomNumberForColor()].toString();
+    }
+
+    public int getRandomNumberForColor() {
+        return random.nextInt(Colors.values().length);
     }
```

Comment:
```suggestion
    private int getRandomNumberForColor() {
        return random.nextInt(Colors.values().length);
    }
```

Лучше делать вспомогательные методы приватными